src/frontend/context/SecurityContext.tsx

```diff
@@ -20,13 +20,15 @@ type SecurityContextType = {
   setAuthValues: (val: AuthSetters) => void;
   authenticate: (val: string | null, validateOnly?: boolean) => boolean;
   authState: AuthState;
+  setToUnauthenticated: () => void;
```

Comment:
See comment in AppContainerWrapper, I think all this logic can be contained in this context, without needing to expose this API.